test/support/test_helpers.ex

```diff
@@ -258,6 +258,13 @@ defmodule Oli.TestHelpers do
     {:ok, conn: conn, user: user}
   end
 
+  def guest_conn(%{conn: conn}) do
+    guest = user_fixture(%{guest: true})
+    conn = Pow.Plug.assign_current_user(conn, guest, OliWeb.Pow.PowHelpers.get_pow_config(:user))
+
+    {:ok, conn: conn, guest: guest}
+  end
```

Comment:
We could have used `user_conn/2` passing `attrs = %{guest: true}`, but I think it is handy to have it as a separate test helper function, thanks 😄 🙌